Lib/pathlib/_local.py

```diff
@@ -824,7 +823,9 @@ def rmdir(self):
         """
         os.rmdir(self)
 
-    _rmtree = shutil.rmtree
+    def _rmtree(self):
+        import shutil
+        shutil.rmtree(path=self)
```

Comment:
I wouldn't count on the keyword argument `path` and think it is fine to have `shutil.rmtree(self)`.

Reply:
What do you mean by "wouldn't count". I quite like that the keyword makes it clear what is happening, and it reminds me that `self` is actually a path type here.

Reply:
The pathlib codebase usually doesn't do this when calling low-level functions, and I think consistency with the surrounding code is probably more important.